packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -24,6 +24,7 @@
                 :id="`${uniqueId}`"
                 :attributes="$attrs"
                 :type="normalisedInputType"
+                :value="value"
```

Comment:
Could this just be set through the `$attrs` object (so no need for the prop)?

Reply:
Potentially, but personally I find that massively reduces readability